packages/e2e/src/App.tsx

```diff
@@ -40,9 +45,9 @@ const App = () => {
         onStartPayment={() => {
           console.log('onStartPayment');
         }}
-        supportedCurrencies={['USDC', 'SOL']}
-        // paymentType={PaymentRequestType.PAYSTREAM}
-        // totalAmount={0.01} // @TODO bug when also has normalizedPrice
+        supportedCurrencies={undefined}
+        paymentType={PaymentRequestType.PAYLINK}
+        totalAmount={undefined}
```

Comment:
no need to provide undefied

Reply:
@martinavagyan this is the development file, please don't review this file. thanks